src/main/puzzle/puzzle-score.gd

```diff
@@ -94,7 +98,11 @@ help/hurt things like their box rank or combo rank. It's mostly to put on a show
 wasn't wasted, if they built a lot of boxes they didn't clear.
 """
 func trigger_finish() -> void:
+	if game_ended:
```

Comment:
is this check necessary? it feels redundant since the only place calling this function checks `PuzzleScore.game_active`

Reply:
Removed redundant check -- if this causes any bugs we'll find out over the next few days